api/envoy/config/listener/v3/listener.proto

```diff
@@ -358,3 +362,8 @@ message Listener {
   // :ref:`global_downstream_max_connections <config_overload_manager_limiting_connections>`.
   bool ignore_global_conn_limit = 31;
 }
+
+message Fcds{
+  string fcds_name = 1 [(validate.rules).string.min_len = 1];
+  envoy.config.core.v3.ConfigSource config_source = 2 [(validate.rules).message.required = true];
+}
```

Comment:
This should probably be called ListenerFCDS, to differentiate between listener and http filter chains.